main.html

```diff
@@ -0,0 +1,128 @@
+                        </li>
+                    </ul>
+                </div>
+            </nav>
+        </div>
+    </header>
+    <section class="hero">
+        <div class="container">
+            <aside class="aside__contact">
+                <ul class="aside__content">
+                    <li class="aside__item">+04 070 991 598</li>
+                    <li class="aside__item">wild@life.com</li>
+                    <li class="aside__item">Wild Avenue, San Francisco, CA</li>
+                </ul>
+            </aside>
+            <div class="hero__inner">
+                <p class="hero__heading">we teach to protect our </p>
+                <h1 class="hero__title">planet</h1>
+                <p class="hero__content">Our planet is precious and vital for our beautiful lives,if we don't take care
+                    of it it will dissapear along with us.</p>
+                <p class="hero__undernote">*Please take note that your actions matter more than you think.</p>
+                <div class="hero__CTA">
+                    <a class="hero__button" href="#">Find out more</a>
+                    <a class="hero__button--transparent" href="#">Play video</a>
+                </div>
+            </div>
+    </section>
+    <div class="about__container">
+        <div class="about__mission">
+            <h1 class="about__title">Our Mission</h1>
+            <div class="about__content">Our wildlife is important to us
+                if we don’t take care of it
+                maybe one day will turn
+                against us and we will die.
+            </div>
+            <div class="about__content">Our wildlife is important to us
+                if we don’t take care of it
+                maybe one day will turn
+                against us and we will die.
+            </div>
+            <div class="about__content">Our wildlife is important to us
+                if we don’t take care of it
+                maybe one day will turn
+                against us and we will die.
+            </div>
+        </div>
+        <div class="about__gallery">
+            <div class="about__flex">
+                <div class="about__card">
+                    <div class="card__content">
+                        <h1 class="card__title">Wildlife</h1>
+                        <p class="card__info">
+                            Our wildlife is important to us
+                            if we don’t take care of it
+                            maybe one day will turn
+                            against us and we will die.
+                        </p>
+                        <a class="hero__button--card" href="#">Find out more</a>
+                    </div>
+                </div>
+                <div class="flex__big">
+                    <div class="flex__sheep">
+                        <img class="photo__sheep" alt="view of sheeps" src="about-sheep.jpg">

```

Comment:
Usually you style images the same. So no point in having three different classes for the same styling.
You could use .image instead of .photo__sheep. Also, you'd have to have a parent .photo somewhere.